README.md

```diff
@@ -12,25 +12,30 @@
 
 - [Manual](http://juliareach.github.io/SX.jl/latest/)
 - [Contributing](https://juliareach.github.io/SX.jl/latest/about.html#Contributing-1)
+- [Release notes](https://github.com/JuliaReach/SX.jl/releases)
+- [Release notes of the development version](https://github.com/JuliaReach/SX.jl/wiki/Release-log-tracker)
 
 ## Dependencies
 
 This package relies on the interfaces defined in [HybridSystems.jl](https://github.com/blegat/HybridSystems.jl) and [MathematicalSystems.jl](https://github.com/JuliaReach/MathematicalSystems.jl). To handle XML files, we use [EzXML.jl](https://github.com/bicycle1885/EzXML.jl). Symbolic algebraic manipulations are performed with [SymEngine](https://github.com/symengine/SymEngine.jl). 
 
 ## Installing
 
-This package requires Julia v0.6 or later. Refer to the [official documentation](https://julialang.org/downloads)
-on how to install and run Julia in your system.
+This package requires Julia v1.0 or later. Refer to the
+[official documentation](https://julialang.org/downloads) on how to install and
+run Julia in your system.
 
-To install the package SX, use the following command inside Julia's REPL:
+To install the package `SX`, use the following command inside Julia's REPL:
 
 ```julia
+using Pkg
 Pkg.clone("https://github.com/JuliaReach/SX.jl")
 ```
 
 ## Updating
 
-To checkout the latest version, do
+To checkout the latest (development) version, do:
+
 ```julia
-Pkg.checkout("SX")
-````
+Pkg.add(PackageSpec(url="https://github.com/JuliaReach/SX.jl", rev="master"))
```

Comment:
Is `rev="master"` not the default?

Reply:
Hmm i don't know, probably yes? This use is one of the examples in the docstring,

```julia
  Examples
  ≡≡≡≡≡≡≡≡≡≡

  Pkg.add("Example") # Add a package from registry
  Pkg.add(PackageSpec(name="Example", version="0.3")) # Specify version
  Pkg.add(PackageSpec(url="https://github.com/JuliaLang/Example.jl", rev="master")) # From url
  Pkg.add(PackageSpec(url="/remote/mycompany/juliapackages/OurPackage"))` # From path (has to be a gitrepo)
```